cmd/kops/create_cluster.go

```diff
@@ -170,6 +172,7 @@ func NewCmdCreateCluster(f *util.Factory, out io.Writer) *cobra.Command {
 	cmd.Flags().StringVar(&options.Image, "image", options.Image, "Image to use")
 
 	cmd.Flags().StringVar(&options.Networking, "networking", "kubenet", "Networking mode to use.  kubenet (default), classic, external, kopeio-vxlan, weave, flannel, calico, canal.")
+	cmd.Flags().BoolVar(&options.NetworkEncrypt, "network-encrypt", options.NetworkEncrypt, "Weave network support encryption between nodes. Password stored in etcd as k8s secret weave-pass.")
```

Comment:
secret for weave is secret in k8s, if i correct understand - secret in kops is different thing. is not ?